arbre.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.description = %q{Arbre makes it easy to generate HTML directly in Ruby}
   s.license     = "MIT"
 
-  s.files         = `git ls-files LICENSE docs lib`.split("\n")
+  s.files         = Dir['docs/**/*', 'lib/**/*', 'LICENSE'].reject { |f| File.directory?(f) }.sort
```

Comment:
The sorting is unnecessary, I just added on both sides on my equality to make sure that the lists had the same files but I don't think it's needed.